main.go

```diff
@@ -133,7 +133,8 @@ func executeHandler(event *corev2.Event) error {
 			name = fmt.Sprintf("%s.%s", handlerConfig.Prefix, name)
 		}
 
-		sender.SendMetric(name, point.Value, point.Timestamp, event.Entity.Name, tags)
+		err = sender.SendMetric(name, point.Value, point.Timestamp, event.Entity.Name, tags)
+		log.Printf("error sending metric: %s", err)
```

Comment:
surely this should have a nil check :)

Reply:
Wow I guess I forgot how to code in a week! Done ✅